proseco/guide.py

```diff
@@ -453,13 +558,21 @@ def get_ax_range(r, extent):
         rminus, rplus = get_ax_range(cand['row'], row_extent)
         cminus, cplus = get_ax_range(cand['col'], col_extent)
         pix = np.array(dark.aca[rminus:rplus, cminus:cplus])
-        pixmax = max(np.max(bin2x2(pix)),
-                     np.max(bin2x2(pix[1:-1])),
-                     np.max(bin2x2(pix[:, 1:-1])),
-                     np.max(bin2x2(pix[1:-1, 1:-1])))
+        pixmax = 0
+        max_r = None
+        max_c = None
+        for i in range(0, pix.shape[0] - 1):
+            for j in range(0, pix.shape[1] - 1):
```

Comment:
This double-loop over the pixels is going to be slower than bin2x2; I just didn't have time to figure out the 4 different indexes into the 2x2 bins sums if I wanted the pixel location of the warmest pixel 2x2.

Reply:
Can you do timing for a few obsids with and without this PR?  Just make sure it isn't totally blowing up.

Reply:
It looks like the densest field takes 20% longer with this PR (but I didn't isolate this code).  Other fields look 5 to 10% longer.  Is that "totally blowing up"?

Should I just add the more verbose logging of what happened to all the candidates as an option?  It isn't clear to me how much we need this output all the time and how much we could just generate it later from the archived inputs.

Reply:
I think I've fixed the binned max code so there's no performance impact.  Will look at the pickle sizes next.